src/utilities/historyUtils.js

```diff
@@ -13,4 +13,32 @@ const setHistory = async () => {
     console.error(error);
   }
 };
-export { setHistory };
+const addToHistory = async (video, dispatch) => {
+  try {
+    const { data } = await axios.post(
+      history,
+      { video },
+      {
+        headers: {
+          authorization: localStorage.getItem("token"),
+        },
```

Comment:
store token into your auth context if you made it , avoid to use like that 
other wise you can also make a one function which return token